dev-requirements.txt

```diff
@@ -8,3 +8,4 @@ sphinx-rtd-theme~=0.4
 sphinx-autodoc-typehints~=1.10.2
 pytest!=5.2.3
 pytest-cov>=2.8
+readme-renderer==24.0
```

Comment:
```suggestion
readme-renderer~=24.0
```

Or why pin it?

Reply:
Fixed. Although I missed the handy buttons below your suggestion.